runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -9837,6 +9858,27 @@ static TR::Register *inlineDoublePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineDoublePrecisionFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
```

Comment:
Can you combine `inlineSinglePrecisionFPTrg1Src3` and this function into a single function to save .so size?  The code is identical except for the size of the target register.

Reply:
I make them two separate functions because `inlineDoublePrecisionFPTrg1Src2` and `inlineSinglePrecisionFPTrg1Src2` functions. Do you think if it is a good idea to use a `bool` variable to indicate target register size if I combine the functions?

Reply:
Why don't you ask the node for its data type and allocate the appropriate kind of targetRegister accordingly?  The targetRegister is what is set on the node, so it is dictated by the data type.

Reply:
Fixed in f980ee7